ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml

```diff
@@ -189,7 +191,7 @@ presubmits:
           - configMap:
               name: cluster-profile-aws-centos
       - configMap:
-          name: prow-job-cluster-launch-e2e-openshift-ansible
```

Comment:
This seems suspicious

Reply:
Dang, found a bug in prowgen

Reply:
https://github.com/openshift/ci-operator-prowgen/pull/39 would fix that